.changeset/nervous-pandas-film.md

```diff
@@ -0,0 +1,5 @@
+---
+"app-builder-lib": minor
```

Comment:
Not sure if I made a right decision here. The scheme change below is a breaking change...